scripts/popup.js

```diff
@@ -315,6 +315,32 @@ function borrow_books(){
     });
 }
 
+function show_news(){
+    chrome.tabs.query({active: true,currentWindow:true},function(tabs){
+        url=tabs[0].url;
+        tabId=tabs[0].id;
+        chrome.storage.sync.get(['news'],function(event){
+            if(event.news==true){
+                if(url.includes("https://www.huffingtonpost"||"https://www.nytimes"||"https://www.forbes"||"https://www.washingtonpost")){
```

Comment:
Is it possible to make a list with these sites so that its possible to add more sites easily?
Then, you could concatenate the list and produce the string you have now... or maybe find another good way to check.
This code is not very nice and its going to be much uglier when more sites are added.